src/bundle/Resources/public/webcomponents/ez-main-content.html

```diff
@@ -0,0 +1,85 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<!--
+    for extensibility, custom field input need to be loaded before the component
+    that wants to use it. For our own field input, we can import them here
+-->
+<link rel="import" href="../../../assets/ezplatform/ez-field-edit/ez-field-edits.html">
+<link rel="import" href="ez-field-edit-richtext.html">
+
+<style>
+ez-main-content {
+    display: block;
+}
+</style>
+
+<script>
+(function () {
+    // this is where we could detect the markup structure to enhance it (tabs, selection table, ...)
+    class eZMainContent extends Polymer.Element {
+        static get is() {
+            return 'ez-main-content';
+        }
+
+        constructor() {
```

Comment:
and given the Content Edit Form is not yet there, all the method from `constructor` to `_getContentEditForm` can be removed as well